src/components/dialog-test.js

```diff
@@ -411,6 +411,86 @@ describes.realWin('Dialog', (env) => {
       expect(styleDuringInit).to.equal('display: none !important;');
     });
 
+    describe('fade background', () => {
+      it('should not register click event by default', async () => {
+        const old = Dialog.prototype.onGrayPaneClick_;
+        let wasClicked = false;
+        Dialog.prototype.onGrayPaneClick_ = function () {
+          wasClicked = true;
+        };
+
+        dialog = new Dialog(globalDoc, {});
+        const el = dialog.graypane_.getElement();
+        const openedDialog = await dialog.open(NO_ANIMATE);
+        await openedDialog.openView(view);
+
+        el.click();
+
+        expect(wasClicked).to.be.false;
+
+        Dialog.prototype.onGrayPaneClick_ = old;
+      });
+
+      it('should register click event', async () => {
+        const old = Dialog.prototype.onGrayPaneClick_;
+        let wasClicked = false;
+        Dialog.prototype.onGrayPaneClick_ = function () {
+          wasClicked = true;
+        };
+
+        dialog = new Dialog(globalDoc, {}, {}, {isClosable: false});
+        const el = dialog.graypane_.getElement();
+        const openedDialog = await dialog.open(NO_ANIMATE);
+        await openedDialog.openView(view);
+
+        el.click();
+
+        expect(wasClicked).to.be.true;
+
+        Dialog.prototype.onGrayPaneClick_ = old;
+      });
+
+      it('should have a click event that returns false', async () => {
+        dialog = new Dialog(globalDoc, {});
+        expect(dialog.onGrayPaneClick_()).to.be.false;
```

Comment:
Similar here, we should be able to generate a click event and check the behavior, instead of calling a private method.

Reply:
I'm not sure how to check the behavior of return false any other way.  Is there a way to get the return value of a click event?

Reply:
This is also why I have it validating that function is the one being called.

Reply:
Fixed